pom.xml

```diff
@@ -2165,6 +2166,9 @@
               <include>org.eclipse.jetty:jetty-security</include>
               <include>org.eclipse.jetty:jetty-util</include>
               <include>org.eclipse.jetty:jetty-server</include>
+              <include>com.sun.jersey:jersey-core</include>
+              <include>com.sun.jersey:jersey-json</include>
+              <include>com.sun.jersey:jersey-server</include>
```

Comment:
Hey @mccheah - if you look at jetty-server as an example, there are other build changes related to shading that you haven't done here:

1. In the root pom, they should be marked as provided.
2. In core/pom.xml they need to be added in includeArtifactIds.
3. They may also need to be listed in other poms as well due to some compiler bugs.

I'd just go through and look everywhere in the source code you see `jetty-server` and do the same for these 3 artifacts. If it's still not working then, let me know and I can take a look.